Source/common/cel/Evaluator.mm

```diff
@@ -150,27 +154,36 @@
   // Everything else is an error.
   if (bool value; result->GetValue(&value)) {
     if (value) {
-      return {{pbv1::ReturnValue::ALLOWLIST, activation.IsResultCacheable()}};
+      return {{Traits::ALLOWLIST, activation.IsResultCacheable()}};
+    }
+    return {{Traits::BLOCKLIST, activation.IsResultCacheable()}};
+  } else if (int64_t value; result->GetValue(&value)) {
```

Comment:
The logic here has changed slightly - want to make sure it is intentional. Previously, the `ReturnValue_IsValid` check would happen within the `if` conditional check, if that failed, we would fall thru to the next `else if` case.

Now though, the "is valid" check happens within the if-statement, such that if it isn't valid, the next `else if` case isn't evaluated and we'll end up falling through to the error condition.

Is this desired? Should we instead make another function wrapper in CELProtoTraits for `ReturnValue_IsValid` similar to `ReturnValue_descriptor` or `ExecutionContext_descriptor`?